src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -22,6 +22,9 @@ public class GenerateEqualsAndGetHashCodeFromMembersTests : AbstractCSharpCodeAc
         private static readonly TestParameters CSharp6 =
             new TestParameters(parseOptions: TestOptions.Regular.WithLanguageVersion(LanguageVersion.CSharp6));
 
+        private static readonly TestParameters CSharp8 =
```

Comment:
This should be the automatic option if nothing is specified; usually we wouldn't say it so we still have coverage once 8.1 or 9.0 or whatever is a thing.